ncm-metaconfig/src/main/metaconfig/httpd/2.4/tests/profiles/davrods.pan

```diff
@@ -0,0 +1,58 @@
+object template davrods;
+
+include 'metaconfig/httpd/schema';
+
```

Comment:
I think this needs a:
include 'components/metaconfig/config';

here to pass the travis test?